jsBasics/task2.js

```diff
@@ -0,0 +1,7 @@
+let name1 = "John";
+let name2 = "Doe";
+
+console.log ("Привіт,", name1 + " та " + name2);
```

Comment:
Ну можна і так, але краще зробити
```suggestion
console.log ("Привіт," + name1 + " та " + name2);
```